### PR TITLE
Added pagination

### PR DESCRIPTION
**Pull request by miguelToscano**

### CODE REVIEW

File: candid/address_book.did [hunk 1]
```diff
@@ -26,6 +26,7 @@ type operation_response = variant {
 service : {
     "name"           :  () -> (text) query;
     "get_all"        :  () -> (vec address);
+    "get_all_paginated"        :  (offset: opt nat64, limit: opt nat64) -> (variant { Ok: vec address; Err: operation_error });
```
Comment (Nima-Ra) [inline candid/address_book.did:29]: I recommend this:
```suggestion
    "get_all_paginated" :  (offset: opt nat64, limit: opt nat64) -> (opt vec address);
```

File: registries/address_book/Cargo.toml [hunk 2]
```diff
@@ -18,6 +18,7 @@ crc32fast = "1.3.2"
 ic-cdk-macros = "0.3"
 serde_bytes = "0.11.5"
 validator = { version = "0.12", features = ["derive"] }
+tokio = { version = "1.17.0", features = ["full"] }
```
Comment (Nima-Ra) [inline registries/address_book/Cargo.toml:21]: I had used `tokio` for the other registries but when I ran `cargo test` the tests didn't pass successfully. Does this work with the address book canister?

**Review by Nima-Ra — APPROVED** [review]
I have to compile and test it before I post my full review but it looks good to me! 👌

File: candid/address_book.did (hunk 1 above)
Comment (miguelToscano) [reply candid/address_book.did:29]: And returning null in case of bad params?

Comment (miguelToscano) [reply candid/address_book.did:29]: And returning null in case of bad params?

File: registries/address_book/Cargo.toml (hunk 2 above)
Comment (miguelToscano) [reply registries/address_book/Cargo.toml:21]: i remember this also happening to me a few weeks ago but this time it worked, i dont know what changed

Comment (miguelToscano) [reply registries/address_book/Cargo.toml:21]: i remember this also happening to me a few weeks ago but this time it worked, i dont know what changed

**Pull request merged**